ovos_bus_client/session.py

```diff
@@ -215,17 +244,26 @@ def get(message=None):
         sess = SessionManager.default_session
         message = message or dig_for_message()
 
+        # A message exists, get a real session
+        if message:
+            LOG.debug(f"Check for session in message")
+            msg_sess = Session.from_message(message)
+            if msg_sess.session_id in SessionManager.sessions:
```

Comment:
this one was not addressed

if the session is not in sessions it should be added there (it's a new session from the client)

```python
msg_sess = Session.from_message(message)
# below should happen unconditionally
SessionManager.sessions[msg_sess.session_id] = msg_sess
```